packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -157,18 +157,69 @@ class AffinityHost extends Component {
 @addSubscriptions({
   resourceSets: subscribeResourceSets,
 })
+@connectStore({
+  isAdmin,
+})
 class ResourceSetItem extends Component {
   _getResourceSet = createSelector(
     () => this.props.resourceSets,
-    () => this.props.id,
-    (resourceSets, id) =>
-      Object.assign(find(resourceSets, { id }), { type: 'resourceSet' })
+    () => this.props.vm.resourceSet,
+    (resourceSets, resourceSetId) => {
+      let resourceSet
```

Comment:
A variable isn't needed for this case. Simply use a `const` for more readability. 

```js
const resourceSet = find(resourceSets, { id: resourceSetId })
return resourceSet && Object.assign(resourceSet, { type: 'resourceSet' })
```